scheduler/remote_server.py

```diff
@@ -628,6 +461,7 @@ def make_schedule():
             # File has been copied
             print(f"Events triggered: {event_types}]")
             if site_id in path:
+                print("Schedule updated - make_schedule() called.")
```

Comment:
Would this statement make more sense inside the `if all` condition just below it? If the `if all` condition is not met, then it looks like make_schedule() would not be called which would make this message rather confusing.

Reply:
You're right, I'll fix this